modules/Makefile

```diff
@@ -16,7 +16,19 @@
 # limitations under the License.
 
 ifndef CHPL_MAKE_HOME
-export CHPL_MAKE_HOME=$(shell pwd)/..
+    export CHPL_MAKE_HOME=$(shell pwd)/..
+endif
+
+# Generate tags only if $TAGS turned on explicitly, or if $CHPL_DEVELOPER is on
+ifndef TAGS
```

Comment:
I don't think there's a better existing place to put this, but maybe it should be styled like the other ones? i.e. 
```
ifdef CHPL_DEVELOPER
TAGS=1
endif
```

(and note that a command like setting of `TAGS`, will still override this default so I don't _think_ the `ifndef TAGS` is needed)